src/apiUrl.js

```diff
@@ -0,0 +1,12 @@
+export const userDetailsUrl = username => `https://api.imgur.com/3/account/${username}`;
```

Comment:
`https://api.imgur.com/3` put this in a const variable and reuse

Reply:
https://github.com/Codebrahma/imgur-client-app/pull/15/commits/fe0b7057b810bc1350ea639e58cd4856f00dca39
Done !